app/routes/ldu-capacity.js

```diff
@@ -0,0 +1,29 @@
+// TODO: swap in real breadcrumbs call when its completed
```

Comment:
Great job on the reusability of the table creator and workload reports views. Looks like we only need one route now instead of 3. We could have both the organisational unit type be supplied as a URL parameter along with the id such as: `/caseload-capacity/:organisationalUnitType/:id/` and then have all of the reusable functions included in that route.